lib-rpc/ocamlformat_rpc_lib.mli

```diff
@@ -9,70 +9,75 @@
 (*                                                                        *)
 (**************************************************************************)
 
-module type Command_S = sig
-  type t
+module Make (IO : IO.S) : sig
+  module type Command_S = sig
+    type t
 
-  val read_input : Stdlib.in_channel -> t
+    val read_input : IO.ic -> t IO.t
 
-  val to_sexp : t -> Sexplib0.Sexp.t
+    val to_sexp : t -> Csexp.t
 
-  val output : Stdlib.out_channel -> t -> unit
-end
+    val output : IO.oc -> t -> unit IO.t
+  end
 
-module type Client_S = sig
-  type t
+  module type Client_S = sig
```

Comment:
This type is not useful outside of the module, right ? I think it can be inlined into `V.Client` and only the `type t` left. Same for `V.Command`.
Otherwise, that's two way of accessing the same functions.

Reply:
I inlined Client_S but Command_S is used twice.